aspnetcore/log-mon/metrics/metrics/samples/custom-metrics/ContosoMetrics.cs

```diff
@@ -0,0 +1,21 @@
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Diagnostics.Metrics;
+
+// <snippet_ContosoMetrics>
+public class ContosoMetrics
+{
+    private readonly Counter<int> _productSoldCounter;
+
+    public ContosoMetrics(IMeterFactory meterFactory)
+    {
+        var meter = meterFactory.Create("Contoso.Web");
```

Comment:
The original snippet `meterFactory.CreateMeter("Contoso.Web")` did not compile for me.

Reply:
Unrelated to the PR, but I think adding a `dotnet-counters` example to read this metric would be helpful. Similar to https://learn.microsoft.com/en-us/aspnet/core/log-mon/metrics/metrics?view=aspnetcore-8.0#view-metrics-with-dotnet-counters.

The reason I bring this up is because it took me a while to figure out that I had to filter on this string (I was using `contoso.product.sold`).

Reply:
Sure, that would be useful.